### PR TITLE
My Lesson 1 & 2 Task 3 done

### PR DESCRIPTION
**Pull request by iamipheco**
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm

I'm done with my Lesson 1 & 2 Task 3, Please Kindly Check it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+
+- Images/ This directory contains images used on the website.
+
+## How to run/test this project
+
+- ensure you have git installed properly on your machine
+
+- ensure you have node js installed in your machine
+
+- open your terminal and run "git clone "
```
Comment (OgJaykay) [inline README.md]: Add your repo link

**Review by OgJaykay — COMMENTED** [review]
Update your docs folder to html folder

**Comment by OgJaykay** [comment]
Edit and merge
good work👍

**Comment by iamipheco** [comment]
> Edit and merge good work👍

Alright Thanks

**Pull request merged**